package.json

```
@@ -9,7 +9,14 @@
         "src/",
         "config.js"
     ],
-    "repository": "git@github.com:salesforce/sfdx-lwc-jest.git",
+    "repository": {
+        "type": "git",
+        "url": "git+ssh://git@github.com/salesforce/sfdx-lwc-jest.git"
```

Based on the [package.json](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository) you can use the HTTPS scheme for the repository. It's something we are already doing for [other packages](https://github.com/salesforce/eslint-config-lwc/blob/master/package.json#L45).

```suggestion
        "url": "https://github.com/salesforce/sfdx-lwc-jest.git"
```